projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1861,8 +1862,16 @@ public static function wrap_message_in_html_tags( $title, $body, $footer ) {
 			return $body;
 		}
 
-		$template = '';
-		$style    = '';
+		$template       = '';
+		$style          = '';
+		$event          = new Jetpack_Tracks_Event(
+			(object) array(
+				'_en' => 'jetpack_forms_email_open',
+				'_ui' => hash_hmac( 'md5', get_option( 'admin_email' ), JETPACK__VERSION ),
```

Comment:
This hash here is just to have a unique, anonymized string for this field. The secret key used in the hash_hmac method is not that relevant (we're not going to validate it). So, I used JETPACK__VERSION, but I could have used any other constant.

Reply:
Above could be a code comment too, for the next one looking at this?